### DISCUSSION TITLE
Can I define variables and output them?

### DISCUSSION THREAD

**Discussion by mrzzoxo** [vuejs/vitepress#4411] [Q&A] [answered]
I'm editing in config.ts

``` ts
  themeConfig: {

    globalComputed: {
      mail() {
        return 'pony@gmail.com';
      },
      telegram() {
        return 'pony';
      },
    },

  },
```

Then test the output index.md

``` markdown

Email：{{ mail }}
Telegram：{{ telegram }}

```

Why is there no output variable content?

![image](https://github.com/user-attachments/assets/3776ae71-1a48-486d-8e40-355215ef990c)

---
I want to define two variables
Then insert and use it in any .md

The above operation was found in [gemini](https://gemini.google.com), is there something wrong with me? Or is vitepress not supported?

My native language is not English. If I use Google Translate, the expression may not be clear. Please forgive me!

**Comment by brc-dd** [comment, marked as answer]
gemini is hallucinating. Refer https://github.com/vuejs/vitepress/discussions/634#discussioncomment-11317369

`app.config.globalProperties.telegram = 'pony'`
